lib/valkyrie/persistence/postgres/persister.rb

```diff
@@ -31,6 +32,7 @@ def delete(resource:)
       resource
     end
 
+    # (see Valkyrie::Persistence::Memory::Persister#wipe!)
```

Comment:
@tpendragon should this be an '@see' ?

Reply:
Nah, this copies the relevant documentation to this node. Looks nicer in the YARD